### PR TITLE
Update workflow name and badges

### PR DESCRIPTION
**Pull request by klappradla**
- update workflow name
- add badge for CI to README
- update npm badge to a fancier one

Note: we may need to update which workflow is required for PRs as the name changed...

### CODE REVIEW

**Review by planktonic — APPROVED** [review]
Can’t see the results until it’s merged I guess? But looks like some much-needed updates 🎉

**Comment by klappradla** [comment]
> Can’t see the results until it’s merged I guess?

You can in the rich diff:

Screenshot 2022-02-01 at 11 55 02

**Comment by planktonic** [comment]
@klappradla Aha, thanks :D  I’d say maybe it would read better as `Linting & tests | Passed`? (without “run”) But that’s a total nitpick, I’m happy with this as it is ⚡️

**Comment by klappradla** [comment]
> @klappradla Aha, thanks :D I’d say maybe it would read better as `Linting & tests | Passed`? (without “run”) But that’s a total nitpick, I’m happy with this as it is ⚡️

It will be exactly that. That's what my name change in the actions file was for.

File: .github/workflows/run-checks.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-name: Run linting & tests
+name: Linting & tests
```
Comment (planktonic) [inline .github/workflows/run-checks.yml:1]: > (only change after the merge is the name on the label 🤷 )

Aha, now I understand that sentence :+1: Looks great!

**Pull request merged**